pkgs/package-overrides.nix

```diff
@@ -42,23 +42,23 @@ in
               url = "https://github.com/php/php-src/commit/e29922f054639a934f3077190729007896ae244c.patch";
               sha256 = "zC2QE6snAhhA7ItXgrc80WlDVczTlZEzgZsD7AS+gtw=";
             })
+          ] ++
+            lib.optionals (lib.versionOlder prev.php.version "7.4") [
+             (pkgs.fetchpatch {
+              url = "https://github.com/php/php-src/commit/4cc261aa6afca2190b1b74de39c3caa462ec6f0b.patch";
+              sha256 = "11qsdiwj1zmpfc2pgh6nr0sn7qa1nyjg4jwf69cgwnd57qfjcy4k";
+              excludes = [ "ext/dom/tests/bug43364.phpt" "ext/dom/tests/bug80268.phpt" ];
+            })
```

Comment:
The indent does not match here, should be 2 spaces.

Reply:
oops. Going to fix that.

Reply:
Also would be nice to have excludes one per line.

Reply:
Still the closing parens look misaligned.